support/linux/install_dev_9_linux.sh

```diff
@@ -21,6 +21,6 @@ else
   sudo -E addgroup --system hab || true
 fi
 
-sudo -E sh /tmp/install.sh
-sudo -E hab install core/busybox-static core/hab-studio
-sudo -E rm -rf /tmp/install.sh
+sudo sh /tmp/install.sh
+sudo hab install core/busybox-static core/hab-studio
+sudo rm -rf /tmp/install.sh
```

Comment:
Removing the `-E` to prevent having a `HAB_BLDR_URL` set interfering. If we decide we ever need this script to install from somewhere other than the default builder, we can always make it more flexible, but I tend to think YAGNI.